resources/log4j2.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+  <Appenders>
+    <Console name="STDOUT" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d{MM-dd HH:mm:ss} %highlight{%p} %style{%c{2}}{bright} :: %m%n">
```

Comment:
Perhaps out-of-scope, but can we add the year to the logs? Is there a reason we don't have it in there (maybe @camsaul knows)?

Reply:
yep, we can switch that. the file log pattern does include the year, this pattern is only the console

Reply:
I don't know the reasoning for the logging pattern we have, but I can tell you we aren't doing anything magical with the logs. So if you guys think year is useful or want to do anything else then go for it